WalletWasabi.Tests/UnitTests/WabiSabi/Backend/CoordinatorTests.cs

```diff
@@ -74,9 +76,10 @@ public void BanDoubleSpendersTest()
 		var workDir = Common.GetWorkDir();
 		CoordinatorParameters coordinatorParameters = new(workDir);
 		WabiSabiConfig cfg = coordinatorParameters.RuntimeCoordinatorConfig;
-		DoSConfiguration dosConfig = cfg.GetDoSConfiguration();
+		DoSConfiguration dosConfig = cfg.GetDoSConfiguration() with { MinTimeInPrison = TimeSpan.Zero };
 		var coinJoinIdStore = new InMemoryCoinJoinIdStore();
-		using WabiSabiCoordinator coordinator = new(coordinatorParameters, NewMockRpcClient(), coinJoinIdStore, new CoinJoinScriptStore(), new MockHttpClientFactory());
+		var mockRpcClient = new MockRpcClient { Network = Network.Main };
```

Comment:
Small nit, inconsistency: 
L138: `var tx = Network.RegTest.CreateTransaction();` -> RegTest is used here, though it changes nothing, test still passes.
L149: `var mockRpcClient = new MockRpcClient { Network = Network.Main };`

Reply:
I saw this but I decided to do not make unnecessary changes. Before it used `NewMockRpcClient` which uses `Main`. Nothing changed.